test/datatable/demo/data.json

```diff
@@ -66,7 +66,7 @@
     "state": "Rhode Island",
     "street": "2492 Hermiston Lodge",
     "country": "Bangladesh",
-    "login": "Blaise.Dicki",
+    "login": "",
```

Comment:
Why did we remove all the names from this file?

Reply:
Well, it was to give some context for the visual playground: with a login, set a button to connect, without propose to create the account